Dockerfile

```diff
@@ -1,21 +1,21 @@
 FROM node:14 as builder
 
 WORKDIR /calcom
-ARG NEXT_PUBLIC_WEBAPP_URL
+ARG NEXT_PUBLIC_WEBAPP_URL=http://localhost:3000
 ARG NEXT_PUBLIC_APP_URL
 ARG NEXT_PUBLIC_LICENSE_CONSENT
-ARG NEXT_PUBLIC_TELEMETRY_KEY
-ARG NEXTAUTH_SECRET
-ARG CALENDSO_ENCRYPTION_KEY
+ARG NEXT_PUBLIC_TELEMETRY_KEY=js.2pvs2bbpqq1zxna97wcml.oi2jzirnbj1ev4tc57c5r
```

Comment:
The telemetry key isn't required anymore. We switched to an [opt-out variable](https://github.com/calcom/cal.com/blob/6f8d0521f47f63d8d1f3300044a5973f126c37fb/.env.example#L57-L58) but it is completely optional

```suggestion
```